js/pihole/queryads.js

```diff
@@ -1,38 +1,77 @@
+var exact = "";
 function eventsource() {
     var ta = $("#output");
     var domain = $("#domain");
+    var q = $("#quiet");
     if(domain.val().length === 0)
     {
         return;
     }
-    var source = new EventSource("php/queryads.php?domain="+domain.val());
+
+    var quiet = false;
+    if(q.val() === "yes")
+    {
+        quiet = true;
+    }
+
+    if(quiet)
+    {
+        exact = "exact";
+    }
+
+    var host = window.location.host;
+    var source = new EventSource("http://"+host+"/admin/php/queryads.php?domain="+domain.val()+"&"+exact);
```

Comment:
I think that it is cleaner if everything comes form the same domain, e.g. `analytics.google.com`